docs/README.md

```diff
@@ -126,7 +117,11 @@ sail down
 sail down --rmi all -v
 ```
 
-## Working with OAuth <a id="oauth"></a>
+### Updating the Table of Contents (ToC)
+
+The project uses `doctoc` to auto generate the Table of Contents (ToC) of all `*.md` files in the `/docs` folder. To update the ToC locally run `$ npx doctoc ./docs --maxlevel=2`.
```

Comment:
Since we already have an NPM script for this, why not use it directly in the example?